lib/streaming-pipeline.js

```diff
@@ -106,22 +97,35 @@ module.exports = class StreamingPipeline extends Writable {
         }
 
         if (inputSignal.hasStart()) {
-            const outputContentTypes = inputSignal.getStart().getExpectedcontenttypesList();
+            const startSignal = inputSignal.getStart();
+            const outputContentTypes = startSignal.getExpectedcontenttypesList();
+            const inputStreamNames = startSignal.getInputnamesList();
+            const outputStreamNames = startSignal.getOutputnamesList();
             if (this._startReceived) {
                 callback(new RiffError(
                     'error-streaming-too-many-starts',
-                    `start signal has already been received. Rejecting new start signal with content types [${outputContentTypes.join()}]`));
+                    'start signal has already been received. ' +
+                    `Rejecting start signal with: output content types: [${outputContentTypes.join(",")}], ` +
```

Comment:
do we care if outputContentTypes contains a comma? That's going to be quite common.

Reply:
Shall I use a different separator or dump the array as is?